src/map/script.c

```diff
@@ -8056,6 +8056,44 @@ BUILDIN(getguildmember)
 }
 
 /*==========================================
+ * Checks the relationship between 2 guilds.
+ * Return :-
+ * -1 - either of the guild is not exist
+ *  0 - they don't have any relationship
+ *  1 - they are in alliance
+ *  2 - they are in antagonist
+ *------------------------------------------*/
+BUILDIN(checkguildally)
+{
+	struct guild *g1, *g2;
+	int gid1, gid2, i = 0;
+
+	gid1 = script_getnum(st,2);
+	gid2 = script_getnum(st,3);
+	g1 = guild->search(gid1);
+	g2 = guild->search(gid2);
+
+	if (g1 == NULL || g2 == NULL) {
+		ShowWarning("buildin_checkguildally: requesting non-existent guild ID.\n"); //guild->search already tell the guild ID
```

Comment:
AHHH HAHAHA !
`[Warning]: intif: guild noinfo 8`
this warning comes from this script command
`    requestguildinfo .@gid; // <-- this`